service-front/module/Application/view/application/pages/identity_check_failed.twig

```diff
@@ -20,5 +20,5 @@
         {% else %}
             <a href="./how-will-donor-confirm" class="govuk-button">Try a different method</a>
         {% endif %}
-
+    </main>
```

Comment:
I think the correct fix here (and on `identity_check_passed.twig` and `confirm_lpas.twig`) is to remove the opening `<main>` tag above, as you did in the other templates on #116